src/rtp/ntp.c

```diff
@@ -52,7 +59,15 @@ void ntp2unix(struct timeval *tv, const struct ntp_time *ntp)
 }
 
 
-int ntp_time_get(struct ntp_time *ntp)
+/**
+ * Obtain the current wallclock time in NTP and jiffies formats
+ *
+ * @param ntp NTP time
+ * @param jfs Milliseconds since Unix epoch
+ *
+ * @return 0 on success, otherwise error code
+ */
+int get_ntp_time_jiffies(struct ntp_time *ntp, uint64_t *jfs)
```

Comment:
this function now returns the wallclock time in two different formats.

is it possible to return only one format, and convert it somewhere else ?

Reply:
No, because AFAIK there is no format in libre that has nanosecond precision. This is needed to get precise timestamps.